### PR TITLE
init_project

### PR DESCRIPTION
**Pull request by meoconlontonbebong**
init project

### CODE REVIEW

**Comment by phungvankien1997** [comment]
ready

**Comment by phungvankien1997** [comment]
commented

**Comment by phungvankien1997** [comment]
ready

**Comment by phungvankien1997** [comment]
ready

**Comment by meoconlontonbebong** [comment]
ready

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.kienphung.music_52"
```
Comment (DoanVanToan) [inline app/build.gradle:6]:
```suggestion
        applicationId "com.kienphung.music_52"
```

File: app/src/androidTest/java/com/example/kienphung/music_52/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getTargetContext();
+
+        assertEquals("com.example.kienphung.music_52", appContext.getPackageName());
```
Comment (DoanVanToan) [inline app/src/androidTest/java/com/example/kienphung/music_52/ExampleInstrumentedTest.java:24]:
```suggestion
        assertEquals("com.kienphung.music_52", appContext.getPackageName());
```

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml:12]: remove hardcode

File: app/src/test/java/com/example/kienphung/music_52/ExampleUnitTest.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() {
+        assertEquals(4, 2 + 2);
+    }
+}
```
Comment (DoanVanToan) [inline app/src/test/java/com/example/kienphung/music_52/ExampleUnitTest.java:17]: add a new line at end of file

**Review by DoanVanToan — COMMENTED** [review]
add a new line at end of file

**Pull request closed without merging**